syntax = "proto3";

package counter;

service Counter {
    rpc Increment (IncrementRequest) returns (IncrementResponse) {}
}

message IncrementRequest {
    string name = 1;
}

message IncrementResponse {
    int32 count = 1;
}